math/src/main/java/org/apache/mahout/math/AbstractMatrix.java

```diff
@@ -782,13 +782,34 @@ public boolean isAddConstantTime() {
 
   @Override
   public String toString() {
+    int row = 0;
+    int maxRowsToDisplay = 10;
+    int maxColsToDisplay = 20;
```

Comment:
If there are no objections to a max 10x20 matrix to display, or anything else, i will commit this to master and the 0.10.1 branch.